platform/ios/src/MGLMapView.mm

```diff
@@ -194,7 +197,7 @@ @implementation MGLMapView
     MGLAnnotationContextMap _annotationContextsByAnnotationTag;
     /// Tag of the selected annotation. If the user location annotation is selected, this ivar is set to `MGLAnnotationTagNotFound`.
     MGLAnnotationTag _selectedAnnotationTag;
-    NS_MUTABLE_ARRAY_OF(MGLAnnotationView *) *_annotationViews;
+    NS_MUTABLE_DICTIONARY_OF(NSString *, NSMutableArray *) *_annotationViewReuseQueueByIdentifier;
```

Comment:
The NSMutableArray type should be further constrained to `MGLAnnotationView *`, if that’s possible with the macros we’re using.